pkg/analysis/passes/osvscanner/osvscanner.go

```diff
@@ -15,13 +15,34 @@ import (
 )
 
 var (
-	scanningFailure                    = &analysis.Rule{Name: "osv-scanner-failed", Severity: analysis.Warning}
-	scanningParseFailure               = &analysis.Rule{Name: "osv-scanner-parse-failed", Severity: analysis.Warning}
-	scanningSucceeded                  = &analysis.Rule{Name: "osv-scanner-succeeded", Severity: analysis.Warning}
-	osvScannerCriticalSeverityDetected = &analysis.Rule{Name: "osv-scanner-critical-severity-vulnerabilities-detected", Severity: analysis.Warning} // This will be set to Error once stable
-	osvScannerHighSeverityDetected     = &analysis.Rule{Name: "osv-scanner-high-severity-vulnerabilities-detected", Severity: analysis.Warning}
-	osvScannerModerateSeverityDetected = &analysis.Rule{Name: "osv-scanner-moderate-severity-vulnerabilities-detected", Severity: analysis.Warning}
-	osvScannerLowSeverityDetected      = &analysis.Rule{Name: "osv-scanner-low-severity-vulnerabilities-detected", Severity: analysis.Warning}
+	scanningFailure = &analysis.Rule{
+		Name:     "osv-scanner-failed",
+		Severity: analysis.Warning,
+	}
+	scanningParseFailure = &analysis.Rule{
+		Name:     "osv-scanner-parse-failed",
+		Severity: analysis.Warning,
+	}
+	scanningSucceeded = &analysis.Rule{
+		Name:     "osv-scanner-succeeded",
+		Severity: analysis.Warning,
+	}
+	osvScannerCriticalSeverityDetected = &analysis.Rule{
```

Comment:
ran gofmt in this file. the only changes are this and the next rule are now errors